src/transform_shapes/transform_shapes.py

```diff
@@ -0,0 +1,30 @@
+from abc import ABC, abstractmethod
+
+import numpy as np
+import cv2
+
+from src.shapes.point import Point
+
+
+class TransformShape(ABC):
```

Comment:
all the functions here are instance functions (they receive self). hence you don't need to send points because points_list is also part of self. Or you change these to static/class methods

Reply:
TrensformingShape can be an inheritor from shape, and then all future shapes will only inherit TrensformingShape

doing this will also allow for better code organization. a shapes folder and abstract shapes folder inside it for instance (and not a sperate folder with a single file)